examples/filepath/tests/filepath_test.bal

```diff
@@ -17,7 +16,7 @@ public function mockPrint(any|error... s) {
 
 boolean isWindows = system:getEnv("OS") != "";
 
-@test:Config
+@test:Config {}
```

Comment:
Shall we remove this test? Since the output is verified in the build?

Reply:
done